modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractScalaCodegen.java

```diff
@@ -529,6 +528,21 @@ protected String formatIdentifier(String name, boolean capitalized) {
         if (identifier.matches("[a-zA-Z_$][\\w_$]+") && !isReservedWord(identifier)) {
             return identifier;
         }
+
+        // below code block only for scala-sttp4-jsoniter for backward copmatibility
+        if (this instanceof ScalaSttp4JsoniterClientCodegen) {
```

Comment:
@lbialy

fyi. i added this check for backward compatibility (otherwise it would break scala-akka tests)

cc @clasnake (2017/07), @shijinkui (2018/01), @ramzimaalej (2018/03), @chameleon82 (2020/03), @Bouillie (2020/04) @fish86 (2023/06)